features/built_in_matchers/any.feature

```diff
@@ -0,0 +1,61 @@
+Feature: `any` matcher
+
+  Use the `any` matcher to specify that a collection's objects any pass an expected matcher.
```

Comment:
`a collection's objects any pass an expected` makes no sense, consider rewording to something like `any of a collections object's pass an expected`